packages/core/rendering/core-rendering-browser/src/use_app_fixed_viewport.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { useRef } from 'react';
+import { APP_FIXED_VIEWPORT_ID } from '@kbn/core-rendering-browser-internal';
```

Comment:
Core's convention doesn't allow importing from *-internal

Reply:
How should I share this ID then? Should core-rendering-browser-internal import it form core-rendering-browser?